pyproject.toml

```diff
@@ -10,3 +10,6 @@ skip = [
     "migrations",
     "node_modules",
 ]
+
+[tool.dirdoc]
+doc_file = "docs/directory-structure.md"
```

Comment:
lol I just like pyproject.toml and this configurability makes perfect sense

Reply:
Good use of `pyproject.toml`!